llvm/lib/SYCLNativeCPUUtils/PipelineSYCLNativeCPU.cpp

```diff
@@ -30,7 +30,17 @@
 #include "compiler/utils/work_item_loops_pass.h"
 #include "vecz/pass.h"
 #include "vecz/vecz_target_info.h"
+#include "llvm/IR/PassManager.h"
+#include "llvm/Transforms/AggressiveInstCombine/AggressiveInstCombine.h"
 #include "llvm/Transforms/IPO/AlwaysInliner.h"
+#include "llvm/Transforms/Scalar/ADCE.h"
+#include "llvm/Transforms/Scalar/DCE.h"
+#include "llvm/Transforms/Scalar/EarlyCSE.h"
+#include "llvm/Transforms/Scalar/GVN.h"
+#include "llvm/Transforms/Scalar/LowerExpectIntrinsic.h"
+#include "llvm/Transforms/Scalar/SROA.h"
```

Comment:
Some of these new includes are probably not needed, but this can probably be addressed in subsequent PRs

Reply:
Thanks, I've removed the unnecessary includes